partner_multi_relation/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# © 2013-2016 Therp BV <http://therp.nl>
```

Comment:
Atomize these MRs make one for partner_multi_relation  v10 and one for partner_multi_relation_in_tab v10.
Put on your second MR the note that it depends on partner_multi_relation being merged.

Reply:
@gfcapalbo I do not see the need for this, as it is all part of one functional package.

Reply:
ok.